tstest/integration/vms/top_level_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build linux
+
+package vms
+
+import "testing"
+
+// Code generated by tstest/integration/vms/gen/test_codegen.go DO NOT EDIT
```

Comment:
Should this be done in `go:generate` style? I'm not sure exactly what that entails, but we have a couple examples in the tree already (e.g. Cloner)

Reply:
It is!